### PR TITLE
TM-4241, fix search toggling

### PR DESCRIPTION
**Pull request by M00staff**
Seems like the issue was having the `ResultsFilterContainer` use both props that were drilled into it, along with props from Redux global state... no idea the EXACT reason that is an issue though 🤷

Just grabbing `client` from the parent component now and passing it down to each child

image

fixes - https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2449/files

also tested the CDO "Search as User" and it works as well

### CODE REVIEW

File: src/Components/ResultsContainer/ResultsContainer.jsx [hunk 1]
```diff
@@ -195,10 +196,4 @@ export const mapDispatchToProps = dispatch => ({
...
 });
 
-const mapStateToProps = ({
-  clientView: { client },
-}) => ({
-  client,
-});
-
-export default connect(mapStateToProps, mapDispatchToProps)(ResultsContainer);
+export default connect(null, mapDispatchToProps)(ResultsContainer);
```
Comment (M00staff) [inline src/Components/ResultsContainer/ResultsContainer.jsx:199]: changing this back might be unnecessary - as the fix worked without changing it, but did so for consistency

File: src/Components/ResultsFilterContainer/ResultsFilterContainer.jsx [hunk 2]
```diff
@@ -96,10 +96,4 @@ ResultsFilterContainer.defaultProps = {
   client: {},
 };
 
-const mapStateToProps = ({
-  clientView: { client },
-}) => ({
-  client,
-});
-
-export default connect(mapStateToProps, null)(ResultsFilterContainer);
```
Comment (M00staff) [inline src/Components/ResultsFilterContainer/ResultsFilterContainer.jsx:105]: this bit was the culprit - i'd love to hear some ideas as to why

Comment (elizabeth-jimenez) [reply src/Components/ResultsFilterContainer/ResultsFilterContainer.jsx:105]: I wish I knew 😩 we all tried 😁

**Review by elizabeth-jimenez — APPROVED** [review]
Very nice!

**Pull request merged**